service/controller/resource/pvc/local_pvc.go

```diff
@@ -0,0 +1,79 @@
+package pvc
+
+import (
+	"github.com/giantswarm/apiextensions/v3/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/kvm-operator/service/controller/key"
+)
+
+const (
+	LabelMountTag = "mount-tag"
+)
+
+func newLocalPVCs(customObject v1alpha1.KVMConfig, pvsList *corev1.PersistentVolumeList) ([]*corev1.PersistentVolumeClaim, error) {
+	var persistentVolumeClaims []*corev1.PersistentVolumeClaim
+	localStorageClass := LocalStorageClass
+
+	for i, workerKVM := range customObject.Spec.KVM.Workers {
+		for _, hostVolume := range workerKVM.HostVolumes {
+
+			pv := findPV(pvsList, hostVolume.MountTag)
+			if pv == nil {
+				return nil, microerror.Maskf(notFoundError, "mount tag %s is not available. the mount tag does not exist or is already bound to an existing claim", hostVolume.MountTag)
+			}
+
+			persistentVolumeClaim := &corev1.PersistentVolumeClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: key.LocalWorkerPVCName(key.ClusterID(customObject), key.VMNumber(i), hostVolume.MountTag),
+					Labels: map[string]string{
+						key.LegacyLabelCluster: key.ClusterID(customObject),
+						key.LabelCustomer:      key.ClusterCustomer(customObject),
```

Comment:
These two are probably not necessary. `LabelOrganization` already gives us the customer and `LegacyLabelCluster` is, well, legacy.